### PR TITLE
Add UI test for custom composite call control options

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* Add a UI test for setting specific call and callwithchat composite call control options
* Update callwithchat hermetic tests to support qArgs

# Why
Missing test area, came across a bug when locally testing: #2325 

# How Tested
n/a - see updated snapshots

### CODE REVIEW

File: packages/react-composites/tests/browser/call/hermetic/CallControls.test.ts [hunk 1]
```diff
@@ -31,3 +32,29 @@ test.describe('Call composite custom button injection tests', () => {
...
+    serverUrl
+  }) => {
+    const initialState = defaultMockCallAdapterState([defaultMockRemoteParticipant('Paul Bridges')]);
+    const testOptions: CallCompositeOptions = {
+      callControls: {
+        displayType: 'compact',
+        cameraButton: {
+          disabled: true
+        },
+        microphoneButton: true,
```
Comment (prabhjot-msft) [inline packages/react-composites/tests/browser/call/hermetic/CallControls.test.ts:48]: What does `microphoneButton: true` signify as the microphone is disabled as well in the snapshots

File: packages/react-composites/tests/browser/callwithchat/hermetic/CallControls.test.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }) => {
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    const testOptions: CallWithChatCompositeOptions = {
+      callControls: {
+        displayType: 'compact',
+        cameraButton: {
+          disabled: true
+        },
+        endCallButton: false,
+        microphoneButton: true,
```
Comment (prabhjot-msft) [inline packages/react-composites/tests/browser/callwithchat/hermetic/CallControls.test.ts:23]: same as above

File: packages/react-composites/tests/app/call/BaseApp.tsx [hunk 3]
```diff
@@ -46,7 +46,9 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
                 queryArgs.injectParticipantMenuItems ? onFetchParticipantMenuItems : undefined
               }
               options={
-                queryArgs.injectCustomButtons
+                queryArgs.customCallCompositeOptions
+                  ? queryArgs.customCallCompositeOptions
+                  : queryArgs.injectCustomButtons
```
Comment (mgamis-msft) [inline packages/react-composites/tests/app/call/BaseApp.tsx:51]: Not part of this PR but can we remove `injectCustomButtons` query arg. It seems like a less flexible version of `customCallCompositeOptions`.

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/call/hermetic/CallControls.test.ts (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-composites/tests/browser/call/hermetic/CallControls.test.ts:48]: They are simply valid options so including them in the test, we actually have a bug where setting it to true was causing it to become disabled: #2325

**Review by prabhjot-msft — APPROVED** [review]
